Make error

```
$ make resources

string index out of range
/bin/sh: -c: line 0: conditional binary operator expected
/bin/sh: -c: line 0: syntax error near `""'
/bin/sh: -c: line 0: `if [[  != "" ]];   then echo " table found"; 	else aws --endpoint-url=http://:4569 dynamodb create-table --table-name resources --attribute-definitions AttributeName=tacoIdentifier,AttributeType=S 		AttributeName=externalIdentifier,AttributeType=S 			AttributeName=version,AttributeType=N 			--key-schema "AttributeName=tacoIdentifier,KeyType=HASH" 			--provisioned-throughput=ReadCapacityUnits=10,WriteCapacityUnits=10 			--global-secondary-indexes "IndexName=ResourceByExternalIDAndVersion, 			KeySchema=[{AttributeName=externalIdentifier,KeyType=HASH}, 			           {AttributeName=version,KeyType=RANGE}], 				Projection={ProjectionType=ALL}, 				ProvisionedThroughput={ReadCapacityUnits=10,WriteCapacityUnits=10}" ; fi;'
make: *** [table] Error 2
```

It seems better if I set `LOCAL_ENDPOINT_HOST=localhost`.  Should this be the default?

--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@ PROJECT_UNIT_TESTS     =$(shell go list ./... | grep -v test | grep -v db)
 PROJECT_INT_TESTS      =$(shell go list ./db)
 PROJECT_E2E_TESTS      =$(shell go list ./test)
 LOCALSTACK_SERVICES    =dynamodb,s3
-LOCAL_ENDPOINT_HOST    :=${LOCAL_ENDPOINT_HOST}
+LOCAL_ENDPOINT_HOST    ?=localhost
 LOCAL_ENDPOINT         =--endpoint-url=http://${LOCAL_ENDPOINT_HOST}
 DYNAMO_ENDPOINT        =${LOCAL_ENDPOINT}:4569
 S3_ENDPOINT            =${LOCAL_ENDPOINT}:4572

